bayesian/api_v1.py

```diff
@@ -326,6 +326,11 @@ def get(external_request_id):
         if get_request_count(rdb, external_request_id) < 1:
             raise HTTPError(404, "Invalid request ID '{t}'.".format(t=external_request_id))
 
+        graph_agg = retrieve_worker_result(rdb, external_request_id, "GraphAggregatorTask")
+        if graph_agg is not None and 'task_result' in graph_agg:
+            if graph_agg['task_result'] is None:
+                raise HTTPError(500, 'Invalid manifest file(s) received. Please analyse the stack with proper manifests')
```

Comment:
Can we make a more subtle message like `Please submit valid manifest files for stack-analyses`